### PR TITLE
Add patchPscVerification to pscVerificationService

### PR DESCRIPTION
**Pull request by eedwards0**
First PR for IDVA3-1504
Add patchPscVerification to pscVerificationService
Add new unit test for patchPscVerification

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "author": "Moses Wejuli <mwejuli@companieshouse.gov.uk>",
     "license": "MIT",
     "dependencies": {
-        "@companieshouse/api-sdk-node": "^2.0.159",
+        "@companieshouse/api-sdk-node": "^2.0.162",
```
Comment (PhilipGlover) [inline package.json:34]: the change needed is in https://github.com/companieshouse/api-sdk-node/releases/tag/2.0.163

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by PhilipGlover — APPROVED** [review]
- have used this service in the Statement screen and it has successfully updated my Verification resource in the db
```
        const resource = await patchPscVerification(req, req.params.transactionId, req.params.submissionId,
            {
                "verification_details": {
                    "verification_statements": [VerificationStatement.INDIVIDUAL_VERIFIED]
                }
            }
         )
```
- test passes
- looks good to me

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**